test/sql/test_function/T/test_split

```diff
@@ -0,0 +1,10 @@
+-- name: test_split_constant_source_and_delimiter
+select split('测隔试隔试', '');
```

Comment:
should ut cover the tested bugs? as const input may not come out const columns

Reply:
Fine.